From 27df78a47220c749b9852ded61974fb64a194538 Mon Sep 17 00:00:00 2001
From: Benjamin Drung <benjamin.drung@profitbricks.com>
Date: Thu, 28 Jun 2018 12:10:58 +0200
Subject: [PATCH] Fix bad whatis entries in man pages

Each manual page should start with a "NAME" section, which lists the
name and a brief description of the page separated by "\-". The "NAME"
section is parsed by lexgrog and used to generate a database that's
queried by commands like apropos and whatis. This tag indicates that
lexgrog was unable to parse the NAME section of this manual page.

For manual pages that document multiple programs, functions, files, or
other things, the part before "\-" should list each separated by a
comma and a space. Each thing listed must not contain spaces; a man
page for a two-part command like "fs listacl" must use something like
"fs_listacl" in the "NAME" section so that it can be parsed by
lexgrog.

Refer to the lexgrog(1) manual page, the groff_man(7) manual page, and
the groff_mdoc(7) manual page for details.

This fixes the lintian warning manpage-has-bad-whatis-entry.

Signed-off-by: Benjamin Drung <benjamin.drung@profitbricks.com>
---
 libibverbs/man/ibv_create_counters.3.md | 2 +-
 libibverbs/man/ibv_read_counters.3.md   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/libibverbs/man/ibv_create_counters.3.md b/libibverbs/man/ibv_create_counters.3.md
index 392e0a57..dc9f9795 100644
--- a/libibverbs/man/ibv_create_counters.3.md
+++ b/libibverbs/man/ibv_create_counters.3.md
@@ -11,7 +11,7 @@ tagline: Verbs
 
 # NAME
 
-**ibv_create_counters**, **ibv_destroy_counters** -- Create or destroy a counters handle
+**ibv_create_counters**, **ibv_destroy_counters** - Create or destroy a counters handle
 
 # SYNOPSIS
 
diff --git a/libibverbs/man/ibv_read_counters.3.md b/libibverbs/man/ibv_read_counters.3.md
index 806d34ea..3bda2ee2 100644
--- a/libibverbs/man/ibv_read_counters.3.md
+++ b/libibverbs/man/ibv_read_counters.3.md
@@ -10,7 +10,7 @@ title: ibv_read_counters
 
 # NAME
 
-**ibv_read_counters** -- Read counter values
+**ibv_read_counters** - Read counter values
 
 # SYNOPSIS
 
-- 
2.17.1

